src/pfe/portal/modules/Project.js

```diff
@@ -783,13 +786,29 @@ module.exports = class Project {
   /**
    * Resolves the given path against the project's monitor path.
    * The path is unchanged if it is already absolute.
-   * 
+   *
    * @param {String} path
    * @returns {String} an absolute path
    */
   resolveMonitorPath(path) {
     return isAbsolute(path) ? cwUtils.convertFromWindowsDriveLetter(path) : join(this.pathToMonitor, path);
   }
+
+  // Project Link wrappers so they can access the writeInformationFile function
```

Comment:
I'm confused by this, User.js and ProjectList.js also call writeInformationFile() already. (I'm not sure if that's a good thing and whether we should rely on callers to write the file to disk or not really though.)

Reply:
`Links.js` is contained within the Project object whereas `User.js` and `ProjectList.js` contain the project objects meaning `Links.js` cannot access the `writeInformationFile()` function. So i've added a wrapper in `Project.js` to ensure it gets written every time a link is changed.

Reply:
Ok, that makes sense. I missed that difference. We don't want to add back links from links to projects as that might trigger circular refs in the project.in file.
This is much better. ;-)